Release 9.0.0

allen-ciq opened the pull request.

allen-ciq committed:
Release 9.0.0

mwilber-ciq commented on src/views/Home.vue:
```diff
@@ -21,7 +21,7 @@
 			<li>
 				<h3>Multi-chart</h3>
-				<p>Coming soon.</p>
+				<p>Comming soon.</p>
```
Typo. I guess this means it needs to be fixed in the private repo?

allen-ciq replied:
Yes, indeedy.  It looks like the fix was committed during the PR/review for [8.8](https://github.com/ChartIQ/chartiq-vue-app/pull/5) , but never made it back into the private repo.  I'll add the fix to both.

allen-ciq committed:
Fixing typo

Was originally fixed in
https://github.com/ChartIQ/chartiq-vue-app/pull/5/commits/03e1965dcc02bd59d3208727306cd8ce70bfa7c5
but never made it back to the private repo.
